applications/GeoMechanicsApplication/tests/test_pressure_line_element.py

```diff
@@ -0,0 +1,77 @@
+import os
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+class KratosGeoMechanicsTransientPressureLineElementTests(KratosUnittest.TestCase):
+    """
+    This class contains benchmark tests which are checked with the regression on a previously obtained value.
+    """
+    etalon_value1 = -20000.0
+
+    def setUp(self):
+        # Code here will be placed BEFORE every test in this TestCase.
+        pass
+
+    def tearDown(self):
+        # Code here will be placed AFTER every test in this TestCase.
+        pass
+
+    def test_oblique_line_element2D2N(self):
+        test_name = 'test_oblique_line_element2D2N'
+        file_path = test_helper.get_file_path(os.path.join('test_pressure_line_element', test_name))
+        simulation = test_helper.run_kratos(file_path)
+        pressure = test_helper.get_water_pressure(simulation)
+        self.assertAlmostEqual(self.etalon_value1, pressure[2])
```

Comment:
It seems we had similar cases and as far as I remember they were done using a local function to avoid copy-and-paste of the lines. Please could you check it?